addon/ng2/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -9,6 +9,12 @@
     "moduleResolution": "node",
     "outDir": "<%= relativeRootPath %>/dist/out-tsc",
     "sourceMap": true,
-    "target": "es5"
+    "target": "es5",
+    "typeRoots": [
+      "../node_modules/@types"
+    ],
+    "types": [
```

Comment:
We can just add the ref to the tests I suppose. But our webpack pipeline can use different tsconfig as well, so we can do that.